tests/test_PolarBasis2D.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+from unittest import TestCase
+
+from aspire.basis.polar_2d import PolarBasis2D
+
+import os.path
+DATA_DIR = os.path.join(os.path.dirname(__file__), 'saved_test_data')
+
+
+class PolarBasis2DTestCase(TestCase):
+    def setUp(self):
+        self.basis = PolarBasis2D((8, 8), 4, 32)
+
+    def tearDown(self):
+        pass
+
+    def testPolarBasis2DEvaluate_t(self):
+        x = np.load(os.path.join(DATA_DIR, 'fbbasis_coefficients_8_8.npy'))
```

Comment:
Same here. Is there no way to avoid hard-coded test data here? It's a relatively simple basis decomposition, so we could calculate it explicitly for a couple of values.

Reply:
I removed the binary file and included the data directly.